code/src/MoviePoster.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+import "./css/MoviePoster.css"
+
+
+export const MoviePoster = ({ id, title, release_date, poster_path }) => {
+    const IMAGE_BASE_URL = 'https://image.tmdb.org/t/p/w342'
+
+    return (
+        <section className='movie-poster'>
+            <Link key={id} to={`/movies/${title}`}>
```

Comment:
I don't think you need mention "key" if you not mapping.

Reply:
Exactly, good catch Moa :)